README.md

```diff
@@ -57,12 +56,12 @@ See the [Get RAPIDS version picker](https://rapids.ai/start.html) for more OS an
 Compiler requirements:
 
 * `gcc`     version 9.3+
-* `nvcc`    version 11.2+
+* `nvcc`    version 11.4+
 * `cmake`   version 3.26.4+
 
 CUDA/GPU requirements:
 
-* CUDA 11.2+
+* CUDA 11.4+
 * NVIDIA driver 450.51+
```

Comment:
Driver 450 is past EOL. We've discussed dropping driver requirements from RAPIDS docs entirely. What do you think? cc: @raydouglass
```suggestion
```